apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -1473,11 +1472,11 @@ def update_var(
 
         context: Optional :class:`SOMATileDBContext` containing storage parameters, etc.
 
-        platform_config: Platform-specific options used to update this array, provided
-        in the form ``{"tiledb": {"create": {"dataframe_dim_zstd_level": 7}}}``
+        platform_config: Platform-specific options used to update this array, provided in the form
+            ``{"tiledb": {"create": {"dataframe_dim_zstd_level": 7}}}``
```

Comment:
Sphinx grossly misrenders this (even moving words out of the order they were typed here!!!!!) unless there's this indent. Weird but real.